ntbs-service/Properties/ExternalLinks.cs

```diff
@@ -9,5 +9,7 @@ public class ExternalLinks
         public string License { get; set; }
         public string Copyright { get; set; }
         public string ReportingOverview { get; set; }
+        public string ClusterReportBase { get; set; }
+        public string ClusterReportReplacementSymbol { get; set; }
```

Comment:
Do you think that the symbol parameter is necessary? What would be the downside of hardcoding it? (I would go with something a little more obvious like `<CLUSTER_ID>`)

Reply:
No probably not on second thoughts. The symbol parameter has been switched from `{}` to `<CLUSTER_ID>` and is now hard coded